### DISCUSSION TITLE
LazyVim Extras vs Mason

### DISCUSSION THREAD

**Discussion by chasingtheflow** [LazyVim/LazyVim#3741] [Q&A]
Hi, 

There seems to be a bit overlap between some options in LazyVim Extras and Mason. Which method is preferred / could it cause issues if the same thing is enabled in each?

**Comment by folke** [comment]
What do you mean? Lang extras use mason

**Reply by chasingtheflow** [reply]
I guess what I'm trying to understand is for something like eslint which is present as an option in both is it better to enable it from the lazyvim extras interface or is it better to enable it in mason directly or does it not matter?

**Reply by folke** [reply]
If the extra would just enable eslint in mason, then why would we have an extra?
Extras provide **extra** config for a certain language.
It's all in the docs. You can see for yourself what an extra adds.
